apps/api/pkg/models/user.go

```diff
@@ -2,13 +2,20 @@ package models
 
 import "time"
 
-// User represents a user in the system
 type User struct {
-	ID           int64     `json:"id"`
-	TenantID     int64     `json:"tenant_id"`
-	Username     string    `json:"username"`
+	UserId       int64     `json:"user_id"`
+	FirstName    string    `json:"first_name"`
+	LastName     string    `json:"last_name"`
 	Email        string    `json:"email"`
 	PasswordHash string    `json:"password_hash"`
 	CreatedAt    time.Time `json:"created_at"`
 	UpdatedAt    time.Time `json:"updated_at"`
 }
```

Comment:
Why do you need to change names? We can also remove PasswordHash here, we are using firebase for auth